core/src/main/scala/Client.scala

```diff
@@ -14,8 +14,8 @@ import org.http4s.blaze.pipeline.{Command, TailStage, LeafBuilder}
 import org.http4s.blaze.pipeline.stages.SSLStage
 
 import java.net.InetSocketAddress
+import javax.net.ssl.SSLContext
 import java.nio.ByteBuffer
-import javax.net.ssl.{SNIHostName, SNIServerName}
```

Comment:
Ordering issue for this diff?

Reply:
No, I remove the dependencies introduced in previous commit.